.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
delete this file

Reply:
deleted this sir @amanv8060 , kindly check